### DISCUSSION TITLE
personal-development-esfand-1401/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#9] [Announcements]
# personal-development-esfand-1401/

گزارش توسعه مهارتی من در اسفند ۱۴۰۱.

https://aprd.ir/personal-development-esfand-1401/

**Comment by masoudnasrollahi** [comment]
سلام آقای پورمند،
نوروزتان پیروز.
* بنده قبلا خودم را داخل تلگرام معرفی کرده‌ام. ([+](https://t.me/masooudn))
* خیلی خرسندم از اینکه اکانت فعال شما را داخل متمم دیدم و سپس سر از اینجا درآوردم.
* مرتبط با عنوان اول گزارشتان، پیشنهادم انداختن نگاهی به «بحران تکرارپذیری در علم» است؛ فکر می‌کنم هم جذاب و هم روشنگر باشد، باعث کمی افزایش وسعت دید شود. ([+](https://t.me/evophilosophy/318), [+](http://method.rihu.ac.ir/article_1845.html))

* این دومین وبلاگی‌ست که من می‌بینم به کمک github ساخته شده؛ لطفا در مطلبی درباره‌ی راه‌اندازی آن توضیح دهید. متاسفانه هر بار به بهانه‌ای دست‌وپا کردن وبلاگ را به تعویق انداخته‌ام. راهنمایی شما می‌تواند تسهیل‌گر باشد.

**Reply by pourmand1376** [reply]
سلام جناب نصراللهی، عید شما هم مبارک باشه. امیدوارم که سال خوبی داشته باشید. 

خیلی ممنون از پادکستی که معرفی کردید. هر دو را گوش دادم. خیلی مفید بود. همچنین مقاله این پادکست خود آقای هادی صمدی نوشته بودند. 

قبلاً احساس می‌کردم اکثر مقالات ماشین لرنینگ به درد نخور هستند اما اینطور که به نظر میاد این مسئله عمومی هست. 

در مورد راه اندازی سایت هم چشم. به زودی مطلبی درباره راه اندازی وب‌سایتی شبیه خودم می‌نویسم. 

علی الحساب جستجویی کردم و دیدم مطلب مشابهی نوشته شده.  [(+)](https://virgool.io/@parsiya/farsi-blog-github-pages-tb8lglazafvx)

احتمالا بتونید با این آموزش با دامنه‌های غیر .ir سایت راه اندازی کنید ولی با دامنه ir احتمالا نه. 

برای ir نیاز به سرویس کمکی در کنار github مثل netlify یا cloudflare هست که توضیح خواهم داد.
